src/main/java/cubicchunks/server/chunkio/IONbtWriter.java

```diff
@@ -55,6 +58,8 @@
 @MethodsReturnNonnullByDefault
 @ParametersAreNonnullByDefault
 class IONbtWriter {
+    
+    private static final int MINECRAFT_CHUNK_DATA_VERSION = 819;
```

Comment:
Would it be possible to create some automated test to make sure that we see when vanilla value changes? Maybe by creating new AnvilChunkLoader and calling `saveChunk` with null chunk. It will crash but only after version is already there.